presto-main/src/main/java/com/facebook/presto/operator/aggregation/differentialentropy/DifferentialEntropyStateFactory.java

```diff
@@ -104,9 +104,6 @@ public DifferentialEntropyStateStrategy getStrategy()
         @Override
         public long getEstimatedSize()
         {
-            if (strategy == null) {
```

Comment:
I have not read the whole PR but why do we remove this null check? `strategy` is set in `setStrategy` so this check seems legit.